sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -20,7 +20,10 @@ public interface ISpan {
   @ApiStatus.Internal
   @NotNull
   ISpan startChild(
-      @NotNull String operation, @Nullable String description, @Nullable Date timestamp);
+      @NotNull String operation,
```

Comment:
if you break this api, won't you have to update every call site of `span.startChild`?
The way I did in ruby was to have an optional arg `instrumenter` that defaults to `sentry`.

Reply:
it's marked as internal and is only used in a couple of places by ourselves, but I agree, we should probably add a new method overload here

Reply:
I was explicitly trying to avoid more overloads since this was only used in very few places and it felt cleaner to just add the new param in those places. I did update the call sites. Unfortunately no default params in Java 😢 .

@romtsn you think we need the overload or can we proceed with the changed method signature? At some point we might want to introduce some sort of param object as we did with `startTransaction` and `TransactionOptions` to handle defaults etc.

Reply:
We can proceed I guess